x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/basic_license_essentials_tier/import_rules.ts

```diff
@@ -135,6 +137,30 @@ export default ({ getService }: FtrProviderContext): void => {
         expect(body.errors[0].error.message).toBe('from: Failed to parse date-math expression');
       });
 
+      it('should be able to import rules with defaultable fields', async () => {
+        const defaultableFields: BaseDefaultableFields = {
+          setup: '# some setup markdown',
+        };
+        const ruleToImport = getCustomQueryRuleParams({
+          ...defaultableFields,
+          rule_id: 'rule-1',
+        });
+        const ndjson = combineToNdJson(ruleToImport);
+
+        await securitySolutionApi
```

Comment:
Ditto nit here on the (new to this file) use of `securitySolutionApi`. Consistency is what separates us from the animals.